packages/gestalt/src/Masonry.js

```diff
@@ -64,12 +69,15 @@ type Props<T> = {|
    */
   minCols: number,
   /**
-   * A function that renders the item to be displayed in the grid. This function is passed three props: the item's data, the item's index in the grid, and a flag indicating if Masonry is currently measuring the item.
+   * A function that renders the item you would like displayed in the grid. This function is passed three props: the item's data, the item's index in the grid, and a flag indicating if Masonry is currently measuring the item.
+   *
+   * If present, `heightAdjustment` indicates the number of pixels this item needs to grow/shrink to accommodate a 2-column item in the grid. Items must respond to this prop by adjusting their height or layout issues will occur.
    */
   renderItem: ({|
     +data: T,
     +itemIdx: number,
     +isMeasuring: boolean,
+    +heightAdjustment?: number,
```

Comment:
do we need this atm?

Reply:
Sorry, I'd left a comment explaining and then forgot to submit 🤦 We don't need it yet, so it could be removed. This will be used for future pin leveling work.